superset/db_engine_specs/trino.py

```diff
@@ -188,11 +201,12 @@ def cancel_query(cls, cursor: Any, query: Query, cancel_query_id: str) -> bool:
         :return: True if query cancelled successfully, False otherwise
         """
         try:
+            # Can't use cursor.cancel() because
+            # cursor is new object created in sql_lab.cancel_query
```

Comment:
Not exactly related to your change but I've just realised this method claims to return `True` on success, `False` on failure, but it's actually returning `None` on success, `False` on failure.

Might be a quick win to add a `return True` line while we're here, to fix that.

Reply:
Already has `return True` outside of try-catch block

Reply:
Ah, looks like I'm blind.